src/index.ts

```diff
@@ -1,3 +1,5 @@
 /* istanbul ignore file */
 
 export * from './mento'
+export * from './governance'
+export * from './utils'
```

Comment:
Do you think we want to export all of the function in this file as is or would it be better to wrap them in a way that people can do something like `import { Utils } from '@mento-protocol/mento-sdk'`

Reply:
Thinking about it, we should leave things like this for now. Making this change will require more refactoring of the other classes that make use of the utils. Let's rethink how we are structuring these modules and handling exports when we are done with governance.